extensions/notion/src/search-page.tsx

```diff
@@ -0,0 +1,262 @@
+import {
+  render,
+  ActionPanel,
+  SubmitFormAction,
+  PushAction,
+  Color,
+  Icon,
+  List,
+  Detail,
+  Form,
+  FormValues,
+  OpenInBrowserAction,
+  preferences,
+  showToast,
+  ToastStyle,
+  useNavigation,
+  setLocalStorageItem,
+  getLocalStorageItem,
+  getApplications,
+  getDefaultApplication,
+  closeMainWindow,
+} from '@raycast/api'
+import { useEffect, useState } from 'react'
+import {
+  Database,
+  DatabasePropertie,
+  Page,
+  fetchDatabases,
+  fetchDatabaseProperties,
+  createDatabasePage,
+  searchPages,
```

Comment:
```suggestion
  Page,
  searchPages,
```

Remove unused imports